client/homebrew/pages/archivePage/archivePage.jsx

```diff
@@ -164,6 +164,16 @@ const ArchivePage = createClass({
         );
     },
 
+    validateInput: function (e) {
+        const textInput = e.target
+        const submitButton = document.getElementById('searchButton');
+        if (textInput.value.length > 3) {
```

Comment:
I know I suggested `3`, but two thoughts:  

1)  this is `greater than 3` so it's actually requiring 4, which precludes any searches for "cat" or "dog" which is maybe too tight. But, ...
2)  probably the most important limitation is that the input has `greater than 0` length.  Just has to have something in it.

Reply:
The validation on the Title input that disables/enables the Search button doesn't seem to check if it's valid when the page reloads.  The same valid text is in the field, but because validation only happens at keyUp, it shows invalid--- likely validation method needs to happen at componentDidMount stage.

Same deal if I submit a search with valid Title-- the component reloads, maintains it's valid text, but the Search button is disabled.  Likely same solution as above.

Reply:
Many functions depend on title not being empty, but this should be consistent, i agree

Reply:
Fixed ( i think)